backend/src/main/java/com/google/capstone/GetFavorites.java

```diff
@@ -0,0 +1,42 @@
+package com.google.capstone;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+
+/** Servlet responsible for getting favorites for a given user. */
+@WebServlet(name = "GetFavorites", value = "/api/v1/get-favorites")
+public class GetFavorites extends HttpServlet {
```

Comment:
Do we also need to fetch this information when browsing gallery so that users can un-favorite an artwork?

Reply:
I added the delete favorite feature in a separate PR